packages/model-viewer/src/features/ar.ts

```diff
@@ -308,6 +318,9 @@ configuration or device capabilities');
       if (this.arScale === 'fixed') {
         intentParams += `&resizable=false`;
       }
+      if (this.arPlacement === 'wall') {
+        intentParams += `&enable_vertical_placement=true`;
```

Comment:
@elalish sorry to ask, but where did you find the documentation about this intent parameter? We couldn't find it anywhere on the ARCore SDK docs

Reply:
I have an inside contact 😁. I believe they're working on updating their docs.

Reply:
@elalish , do you know if there is any other option related to vertical placement? Something like "vertical_only" 😅